### ISSUE TITLE
Audio tracks are not caught properly in latest MPC-HC

### ISSUE THREAD

**Issue by AndrewTCEC** [clsid2/mpc-hc#1158]
As far as I know, there are two ways to run audiotracks for video in MPC-HC:
1.Run an integrated audio from a videofile;
2.Create or paste an audio file with the same name and in the same folder as a video that will be caught by the player.
So I am having issues with the 2nd way since MPC-HC 1.9.13 development builds. Any ideas why? Picture with the issue below.
![1](https://user-images.githubusercontent.com/77082125/121785252-6b0be380-cbc1-11eb-902e-189d9b42db63.png)
Player verison MPC-HC 1.9.13.14 (included in K-Lite Codec Pack 16.2.5 Mega, installed it with minor changes to default installation process)
Trying MPC-HC verison 1.9.13.18 you have on your GitHub, results coming shortly...(Success, but the player freezes for me, deleted and reverted).
Will wait for a fix patiently.

**Comment by clsid2** [comment]
I found the video file. Can you upload the ac3 file and subs for me?
http://www.sendspace.com

**Comment by AndrewTCEC** [comment]
Here: https://www.sendspace.com/filegroup/C12F0bfLBuYI1MxNbzhyj6BBayNYmFkLUUWbCEfxQyI
Also some more on that: out of 4 such audio tracks I found on my PC only 3 have a problem, and all problematic tracks have extremely long file name, while one without issues has short file name, hmm...
Also, subtitles are immune to this problem, they are easily caught by the player.
Edit: Update to 1.9.13.19 (integrated with K-Lite Codec Pack 16.2.6) solved the problem looks like! Closing this issue unless it occurs somehow later.
